datadog_checks_dev/datadog_checks/dev/tooling/utils.py

```diff
@@ -16,8 +16,17 @@
 
 # match something like `(#1234)` and return `1234` in a group
 PR_PATTERN = re.compile(r'\(#(\d+)\)')
-
+# match integration's version within the __about__.py module
 VERSION = re.compile(r'__version__ *= *(?:[\'"])(.+?)(?:[\'"])')
+# bits of info telling us we're on a CI platform
+CI_IDENTIFIERS = (
+    'APPVEYOR_',
+    'TRAVIS_',
+)
+
+
+def running_on_ci():
```

Comment:
We can't put this here. This was intentionally placed high up in the root so it can be accessed during testing or e2e `/_env`. Remember, `datadog_checks_dev` is essentially 2 packages with extended dependencies. So if during testing you try to import something past the root you'll be prone to import errors. Nothing in the root imports past the root.

Reply:
I know, it's just not used anywhere aside from the commands, hence the move. And in fact, testing code should never do different things depending on whether they're running.

Reply:
Our joint e2e file `_env` is in the root and will need access to this eventually.

Reply:
Can we please keep it accessible from the root? There are cases where tests would want to know, such as AppVeyor not supporting some Hyper-V things.

Reply:
Can you show me an example?

Reply:
An example would be AppVeyor not supporting virtualization and thus Hyper-V. We would still want to test, say with mocks, but we also want real tests that we/users could run on supported Windows machines that we would skip if on AppVeyor.

Reply:
That would be the test runner job, not the tests themselves. You put a marker on the test that you want to skip on Appveyor and whatever invokes `pytest` would decide whether to run marked tests or not.

Reply:
So you don't feel like use cases that need finer grain control are valid? For example, something like this (we have a bunch) should be its own tox env?

https://github.com/DataDog/integrations-core/blob/17df7b558bc8a832e978b5bc02d4cb54b5f93783/mcache/tests/test_integration.py#L82-L86

Reply:
Thanks for the example, it's indeed very handy to keep short skip conditions like those, commit reverted.